recipes/ragtag/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "RagTag" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f1c5dfdb05efddc0e3fc04f512e764e6d6ab3d7c7c796c8ca42f6ef50d5120a7
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: generic
+
+requirements:
+  host:
+    - intervaltree
+    - numpy
+    - pip
+    - pysam
+    - python >3
+  run:
+    - intervaltree
+    - numpy
+    - pysam
+    - python >3
+
+test:
+  imports:
+    - ragtag_utilities
+
+about:
+  home: "https://github.com/malonge/RagTag"
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "Fast reference-guided genome assembly scaffolding"
+  doc_url: 
```

Comment:
`doc_url: https://github.com/malonge/RagTag/wiki/Usage`

Reply:
fixed in the latest commit - thanks